src/components/Reports/GoalCalculator/Shared/GoalCalculatorContext.tsx

```diff
@@ -15,15 +13,18 @@ import { useSummaryReport } from '../SummaryReport/SummaryReport';
 export type GoalCalculatorType = {
   steps: GoalCalculatorStep[];
   selectedStepId: GoalCalculatorStepEnum;
-  selectedCategoryId: GoalCalculatorCategoryEnum;
   currentStep?: GoalCalculatorStep;
-  currentCategory?: GoalCalculatorCategory;
```

Comment:
so we won't have any category, even on the budget or MPDX goal pages? The MPD Goal has a lot of information which could be good to split up into separate steps.

Reply:
Yes, according to Ryan's latest design all of the categories in a step are all in the same scrollable view:

https://www.figma.com/design/JInpKB7SkTEcJ3pI3dSzYh/MPDX-8585-Staffweb-migrations-to-MPDX?node-id=1874-16949&t=ATcGl4APkGUJ5tCm-0